src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -409,31 +409,38 @@ export class VolumesListTableConfig implements InputTableConf {
                   this.dialogService.confirm("Scrub Pool", message, false, T("Stop Scrub")).subscribe((res) => {
                     if (res) {
                       this.loader.open();
-                      this.rest.delete("storage/volume/" + row1.id + "/scrub/", { body: JSON.stringify({}) }).subscribe(
+                      this.ws.call('pool.scrub', [row1.id, 'STOP']).subscribe(
                         (res) => {
                           this.loader.close();
-                          this.snackBar.open(res.data, 'close', { duration: 5000 });
+                          this.snackBar.open('Stopping the scrub on ' + row1.name + '.', 'close', { duration: 5000 });
```

Comment:
Doesn't this need a T() call?

```suggestion
                          this.snackBar.open(T('Stopping scrub on pool <i>') + row1.name + '</i>.', 'close', { duration: 5000 });
```